core/parsigdb/memory.go

```diff
@@ -165,6 +168,21 @@ func (db *MemDB) store(k key, value core.ParSignedData) ([]core.ParSignedData, b
 	return append([]core.ParSignedData(nil), db.entries[k]...), true, nil
 }
 
+// isThresholdReached checks if threshold is reached or not for given pubkey.
+func (db *MemDB) isThresholdReached(pubkey core.PubKey) bool {
```

Comment:
this is incorrect and will prevent 99% of duties to succeed

Reply:
updated